[2022-07-26 16:57:25,748: INFO/MainProcess] Connected to redis://localhost:6379//
[2022-07-26 16:57:25,774: INFO/MainProcess] mingle: searching for neighbors
[2022-07-26 16:57:26,538: INFO/SpawnPoolWorker-6] child process 8060 calling self.run()
[2022-07-26 16:57:26,570: INFO/SpawnPoolWorker-1] child process 8664 calling self.run()
[2022-07-26 16:57:26,570: INFO/SpawnPoolWorker-2] child process 12020 calling self.run()
[2022-07-26 16:57:26,571: INFO/SpawnPoolWorker-4] child process 6368 calling self.run()
[2022-07-26 16:57:26,617: INFO/SpawnPoolWorker-3] child process 1652 calling self.run()
[2022-07-26 16:57:26,622: INFO/SpawnPoolWorker-5] child process 7432 calling self.run()
[2022-07-26 16:57:26,635: INFO/SpawnPoolWorker-7] child process 6536 calling self.run()
[2022-07-26 16:57:26,650: INFO/SpawnPoolWorker-8] child process 10424 calling self.run()
[2022-07-26 16:57:26,866: INFO/MainProcess] mingle: all alone
[2022-07-26 16:57:26,933: WARNING/MainProcess] C:\Users\Sebastian\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\LocalCache\local-packages\Python37\site-packages\celery\fixups\django.py:204: UserWarning: Using settings.DEBUG leads to a memory
            leak, never use this setting in production environments!
  leak, never use this setting in production environments!''')

[2022-07-26 16:57:26,933: INFO/MainProcess] celery@DESKTOP-CTQTAHN ready.
